### DISCUSSION TITLE
tquic 回调网络传输状态信息功能有时间计划表吗?

### DISCUSSION THREAD

**Discussion by lion117** [Tencent/tquic#306] [Ideas（新功能的建议和讨论）]
# 背景
1. ubuntuX64
2. simple_client.c simaple_server.c
3. libtquic.a

# 需求
使用tquic替换sctp传输协议.  希望能够动态的获取传输过程中的丢包率, RTT, 吞吐量等信息, 以方便业务层做策略调整. 
由于对RUST语言不熟悉, 暂时不具备改造tquic的能力
xquic默认提供此类接口

# 问题
是否有时间计划表开放传输状态信息?

**Comment by glime-ai** [comment]
TQUIC目前已经提供了一些接口来获取传输过程中的性能指标，例如丢包率、RTT和吞吐量等信息。您可以通过以下方式获取这些信息：

1. **可视化和监控**：TQUIC支持灵活的配置参数和丰富的可观测性，您可以通过这些配置来监控传输状态。
2. **多语言接口**：虽然TQUIC主要是用Rust开发的，但它提供了Rust/C/C++的多语言接口，未来还计划提供Kotlin/Swift等语言的接口，这样可以方便您在不熟悉Rust的情况下使用TQUIC。

关于开放传输状态信息的时间计划，目前没有具体的时间表。如果您有特定的需求或建议，建议您在TQUIC的GitHub讨论区提出，这样开发团队可以更好地了解用户需求并进行相应的改进。

您可以参考以下文档获取更多信息：
- [TQUIC文档](https://tquic.net/zh/docs/intro)
- [TQUIC GitHub](https://github.com/tencent/tquic)

This comment was generated by [Glime](https://glimelab.ai/?utm_source=44520fce49ba43798485a628d6928e22&utm_medium=github).

**Comment by iyangsj** [comment]
相关指标信息见 `PathStats`：https://docs.rs/tquic/latest/tquic/struct.PathStats.html
下个版本会提供C语言接口

**Reply by lion117** [reply]
下个版本预计是什么时候? 
可预估吗?

**Reply by iyangsj** [reply]
TQUIC v0.15.0版本，没有固定发布时间，预计7月底

**Reply by lion117** [reply]
值得期待
